### PR TITLE
Event detail template

### PR DESCRIPTION
**Pull request by TerryHCollins**
This is the template for Events that will allow the client to easily add new events in the future.

### CODE REVIEW

File: css/values.css [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+}
+/* Adds a background color in lieu of those rectangle shapes */
+
+.tealback {
+	margin-top: 20px;
+	background-color: var(--color-teal);
+}
+/* Adds a background color in lieu of those rectangle shapes */
+
+.support {
```
Comment (maxharrisnet) [inline css/values.css:107]: Spacing and text sizing on this section is off on desktop. Looks good on mobile though. 

Screen Shot 2024-01-24 at 8 01 22 AM

File: event-detail.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+			href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@400;500;700&display=swap"
+			rel="stylesheet"
+		/>
+		<link rel="stylesheet" href="css/normalize.css" />
+		<link rel="stylesheet" href="css/global.css" />
+		<!-- In a new branch, rename the page.css file adding the name of that actual page (Example: home.css) -->
+		<link rel="stylesheet" href="css/event-detail.css" />
+	</head>
+	<body>
+		<div class="template-wrapper">
```
Comment (maxharrisnet) [inline event-detail.html:144]: Please add the header here. It's in index.html.

File: event-detail.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+				<div class="template-three-img">
+					<img
+						src="./assets/images/pages/event-template/wimw-three.jpg"
+						alt="Digital Stewardship "
+					/>
+				</div>
+			</div>
+		</div>
+	</div>
+	</body>
```
Comment (maxharrisnet) [inline event-detail.html:361]: Please add the footer here. It's in index.html.

File: css/event-detail.css [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+}
+
+.template-wrapper img {
+	align-items: center;
+}
+
+.template-main-header {
+	text-align: center;
+	color: var(--color-onyx);
+	padding: 0 20px 0px 20px;
```
Comment (maxharrisnet) [inline css/event-detail.css]: There are several places in this file where we can replace the values with variables.

File: css/event-detail.css [hunk 5]
```diff
@@ -0,0 +1,91 @@
+/* everything is wonky - on my screen it cuts off on the left - on Gurjeets screen it cuts off on the right or works fine. Looks fine at size 390 */
+html {
+	box-sizing: border-box;
+}
+
+.template-wrapper {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	/* justify-content: center; */
```
Comment (maxharrisnet) [inline css/event-detail.css]: Please delete comment.

File: css/values.css [hunk 6]
```diff
@@ -0,0 +1,248 @@
...
+	border: none;
+	border-radius: 10px;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	text-align: center;
+	margin: 0px 20px;
+}
+.card h4 {
+	margin: 20px 0px 0px 0px;
```
Comment (maxharrisnet) [inline css/values.css]: Please use variables throughout this file where possible.

**Review by maxharrisnet — APPROVED** [review]
This is looking really good. We need the header and footer, the support section needs a little work on desktop, and some values can be replaced by variables in CSS.

**Review by maxharrisnet — CHANGES_REQUESTED** [review]
See comments (header, footer, support section, variables)

File: css/event-detail.css (hunk 5 above)
Comment (TerryHCollins) [reply css/event-detail.css]: done

File: css/event-detail.css (hunk 4 above)
Comment (TerryHCollins) [reply css/event-detail.css]: done

**Pull request merged**